### ISSUE TITLE
Same contacts displayed multiple times on the contacts screen in the signal app

### ISSUE THREAD

**Issue by marcobullin** [WhisperSystems/Signal-iOS#1003]
If your iPhone contact has i.e. a phone number and two email addresses it will be displayed three times in the signal app.

So my contact list in the signal app looks something like:

Fritz Foo
Max Bar
Max Bar
Claus Baz
Claus Baz
Claus Baz

Cheers Marco Bullin

**Comment by FredericJacobs** [comment]
Does he have multiple phone numbers?

**Comment by kunickiaj** [comment]
I experience this as well. 

For me its not due to the fact that a contact has multiple numbers (they have 1) but because they appear in multiple address books (e.g. google contacts, iCloud contacts, etc).

In the iOS Contacts app they appear only once because both entries are linked. At the bottom of the contact details you can see the two "real" contact entries the person is linked to and under which accounts (e.g. iCloud + gmail)

**Comment by FredericJacobs** [comment]
Duplicate of #720
